src/chat.rs

```diff
@@ -1907,6 +1907,71 @@ pub fn forward_msgs(context: &Context, msg_ids: &[MsgId], chat_id: u32) -> Resul
     Ok(())
 }
 
+pub fn get_info_json(context: &Context, chat_id: u32) -> Result<String, Error> {
+    let chat = Chat::load_from_db(context, chat_id).unwrap();
+
+    // ToDo: 
+    // - [x] id
+    // - [x] type
+    // - [x] name
+    // - [x] archived
+    // - [x] color
+    // - [x] profileImage
+    // - [x] subtitle
+    // - [x] draft,
+    // - [ ] deaddrop,
+    // - [ ] summary,
+    // - [ ] lastUpdated,
+    // - [ ] freshMessageCounter,
+    // - [ ] email
+
+    let profile_image = match chat.get_profile_image(context) {
+        Some(path) => path.into_os_string().into_string().unwrap(),
+        None => "".to_string()
+    };
+
+    let draft = match get_draft(context, chat_id) {
+        Ok(message) => {
+            match message {
+              Some(m) => m.text.unwrap_or("".to_string())   ,
+              None => "".to_string()
+            } 
+        },
+        Err(_) => "".to_string()
+    };
+      
+    let s = format!("{{\
```

Comment:
please, please use serde + serde-json for this

Reply:
Thanks, the json! macro is quite nice.